tests/stubtest_whitelists/linux.txt

```diff
@@ -10,6 +10,7 @@ grp.getgrgid
 grp.struct_group._asdict
 grp.struct_group._make
 grp.struct_group._replace
+mmap.MADV_SOFT_OFFLINE  # Not defined in some linuxes
```

Comment:
Is this needed for CI or for running stubtest on your computer? I think `.github/workflows/stubtest-unused-whitelist.yml` will delete this automatically if the CI doesn't need it.

Reply:
Also, `MADV_SOFT_OFFLINE` is new on Linux 2.6.33 (see `madvise(2)` manual page), so unless something hasn't been updated for the past 15 years, this shouldn't be a problem.